### PR TITLE
Update the REPL document

### PR DESCRIPTION
**Pull request by shendiaomo**
Update the enhancement in #1248 to the REPL document.

### CODE REVIEW

File: doc/run/repl.md [hunk 1]
```diff
@@ -78,11 +79,40 @@ sqlflow> SELECT * from iris.predict limit 3;
...
+## Command Line Options
 
 |             Option                      | Description |
 |-----------------------------------------|-------------|
 | -e \<quoted-query-string\>              | Execute from command line without entering interactive mode. e.g. <br>`-e "SELECT * FROM iris.train TRAIN DNNClassifier..." `<br>does the same thing as the training example above.|
 | -f \<filename\>                         | Execute from file without entering interactive mode. e.g. <br>`-f ./my_sqlflow.sql`<br>does the same thing as<br>`< ./my_sqlflow.sql` and `cat ./my_sqlflow.sql \| REPL...` |
 | -model_dir \<local-directory\>          | Save model to a local directory. e.g. `-model_dir "./models/"` |
 | -datasource \<database-connection-url\> | Connect to the specified database. e.g. `-datasource "mysql://root:root@tcp(host.docker.internal:3306)/" ` |
+
+## Keyboard Shortcuts
```
Comment (typhoonzero) [inline doc/run/repl.md:92]: This section is empty

Comment (shendiaomo) [reply doc/run/repl.md:92]: It's a level 2 heading that contains a couple of level 3 headings.

Comment (wangkuiyi) [reply doc/run/repl.md:92]: I fixed the comment by git push directly.

**Review by wangkuiyi — APPROVED** [review]
I git-committed to fix the comment by @typhoonzero .

**Pull request merged**